File: 30-Days-of-Code/Day-2-Operators/Elixir/solution.ex
mealCost = String.to_float(String.trim(IO.gets("")))
tipPercent = String.to_integer(String.trim(IO.gets("")))
taxPercent = String.to_integer(String.trim(IO.gets("")))

tip = mealCost * (tipPercent / 100)
tax = mealCost * (taxPercent / 100)
totalCost = mealCost + tip + tax

roundedTotalCost = round(totalCost)

IO.puts("The total meal cost is #{roundedTotalCost} dollars.")

File: 30-Days-of-Code/Day-2-Operators/Ruby/solution.rb
mealCost = 0.0
tipPercent = 0
taxPercent = 0

mealCost = gets.to_f
tipPercent = gets.to_i
taxPercent = gets.to_i

tip = mealCost * (tipPercent / 100.0)
tax = mealCost * (taxPercent / 100.0)
totalCost = mealCost + tip + tax

roundedTotalCost = totalCost.round

printf "The total meal cost is %d dollars.\n", roundedTotalCost
